packages/manager/src/features/components/PlansPanel/PlanInformation.tsx

```diff
@@ -68,6 +69,23 @@ export const PlanInformation = (props: PlanInformationProps) => {
               </Typography>
             </Notice>
           )}
+          {showTransferBanner && (
+            <Notice spacingBottom={8} variant="warning">
+              <Typography
+                fontFamily={(theme: Theme) => theme.font.bold}
+                fontSize="1rem"
+              >
+                Some plans do not include bundled network transfer. If the
+                transfer allotment is 0, all outbound network transfer is
+                subject to standard charges.
+                <br />
+                <Link to="https://techdocs.akamai.com/cloud-computing/docs/network-transfer-usage-and-costs">
+                  Learn more about transfer costs.
```

Comment:
(extra period showing up in copy)
```suggestion
                  Learn more about transfer costs
```

Reply:
fixed!